src/main/resources/static/templates/CSRFVulnerability/LEVEL_5/CSRFVulnerability.css

```diff
@@ -0,0 +1,63 @@
+#CSRFVulnerability {
+    color: black;
+    text-align: center;
+}
+
+#requestinput {
+    display: flex;
+    flex-direction: column;
+}
+
+#heading {
+	font-size: 20px;
+}
+
+#input {
+	font-size: 15px;
+}
+
+#note {
+    font-size: 15px;
+    margin: 10px;
+}
+
+#lgin {
+    color: #09C700
+}
+
+#lgou {
```

Comment:
I would suggest giving full names so that it is easy to understand the functionality.